spec/mongo/crypt_spec.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe Mongo::Crypt do
+  describe '.validate_ffi!' do
+    context 'when ffi is available' do
```

Comment:
Should there be something here to test for the cases where FFI is not installed? Or skip when it isn't?

Reply:
I couldn't come up with a reasonable complex solution for testing this case. We have gem 'ffi' in our standard set of dependencies for testing. So, we would need a dedicated configuration just to test this feature; I think this is an overhead.

Reply:
Ah! That makes sense. I'll trust that you've thought about this a lot more than I have. :) Thanks for the explanation. Maybe a comment in that spec, explaining the situation, would be helpful to future-us?